src/features/groups/components/InviteMemberButton/index.tsx

```diff
@@ -0,0 +1,63 @@
+'use client';
+
+import { LinkIcon, PlusIcon } from '@/assets/images/icons';
+import {
+  Button,
+  DialogBody,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogRoot,
+  DialogTitle,
+  DialogTrigger,
+  Icon,
+} from '@/components/ui';
+
+import { useEffect, useState } from 'react';
+
+export const InviteMemberButton = () => {
+  const [isDialogOpen, setIsDialogOpen] = useState(false);
+  const [isCopyDisabled, setIsCopyDisabled] = useState(false);
+
+  const handleLinkCopy = () => {
+    setIsCopyDisabled(true);
+  };
+  const handleTextChange = (isClicked: boolean) => {
+    return isClicked ? 'Copied!' : 'Copy Link';
+  };
+  useEffect(() => {
+    setTimeout(() => setIsCopyDisabled(false), 200);
+  }, [isDialogOpen]);
+
+  return (
+    <>
+      <DialogRoot open={isDialogOpen} onOpenChange={setIsDialogOpen}>
+        <DialogTrigger asChild>
+          <Button className="flex justify-center items-center w-12 h-12">
+            <Icon icon={PlusIcon} size={4.5} />
+          </Button>
+        </DialogTrigger>
+        <DialogContent className="h-44 w-11/12">
```

Comment:
To set the dialog's width, maybe better to set it as  `w-[calc(100%-2rem)]`.

It's basically because dialog's width has to be the same with cards!
Card's width is decided with `px-4` in the parent, which means 2rem space from the each side of the screen.

There is a slight unwanted gap between Card's width and dialog's width, when screen width changes.

- With current stying, in `iPhone 14 pro max`
<img width="308" alt="Screenshot 2024-02-07 at 11 29 10" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99339182/125c019d-9602-44a2-a671-f4da04e446ce">

- With `w-[calc(100%-2rem)]` 
<img width="319" alt="Screenshot 2024-02-07 at 11 28 59" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99339182/3e1cd272-353b-41ad-9d9f-fa22dd652fd4">

It might be difficult to see the gap tho 😅  I believe `w-[calc(100%-2rem)]` is a bit more proper solution in here!

Reply:
@kanta1207 
I see the difference, your suggestion is much better, I changed the width to `w-[calc(100%-2rem)]` .

Reply:
It would be better to replace the default width in `DialogContent` with `w-[calc(100%-2rem)]` instead of overriding it. This way the width will be applied to all dialogs implemented throughout the project.

Reply:
Additionally, the height should not be fixed here. The content should determine the dialog's height.

Reply:
<img width="400" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/54383c03-3129-412a-b694-ef77a73ffdd3" />

<img width="400" alt="image" src="https://github.com/nishiki-tech/nishiki-frontend/assets/99148565/c84c9f09-a41c-4418-85c4-68d4b99b9936" />

Could you please ensure that you match the defined fixed sizes in Figma?

Reply:
@nick-y-ito 
Thank you for reviewing my PR. Let me check later, as I am not available. Sorry for inconvenience.
BR
Shy

Reply:
Are you "Shy"? 🤣

Reply:
I'm not anymore🤣

Reply:
@nick-y-ito 
Thank you for the point it out. I added `pb-8` to `DialogBody`